libgraphics/src/legacygradients.cpp

```diff
@@ -951,6 +951,9 @@ MCGradientAffineCombiner *MCGradientFillCreateCombiner(MCGGradientRef p_gradient
 			}
 			break;
 		}
+            
+        default:
+            break;
```

Comment:
See my previous comment about `MCGImageFilter`.  This default case shouldn't be needed.